src/index.ts

```diff
@@ -1,5 +1,6 @@
 export { Application } from './lib/application';
 export { CliApplication } from './lib/cli';
+export * from './lib/stringifiers';
```

Comment:
Does it make sense to export everything here? It seems to make more sense to me to only export `stringifyType` and call the correct stringifier internally.

Reply:
True, I can make that change.